### PR TITLE
Add support for Webex max message length.

### PR DESCRIPTION
**Pull request by xorrkaz**
Webex limits messages to just over 7 KB.  Some alerts might be longer than this.  So truncate the message with ellipses.

While here switch to using the new API URL.

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: LibreNMS/Alert/Transport/Ciscospark.php [hunk 1]
```diff
@@ -19,21 +19,29 @@
...
     {
         $room_id = $this->config['room-id'];
         $token = $this->config['api-token'];
-        $url = 'https://api.ciscospark.com/v1/messages';
+        $url = 'https://webexapis.com/v1/messages';
         $data = [
             'roomId' => $room_id,
         ];
 
+        if (strlen($alert_data['msg']) > Ciscospark::$MAX_MSG_SIZE) {
```
Comment (PipoCanaja) [inline LibreNMS/Alert/Transport/Ciscospark.php:34]: Hi @xorrkaz 
You should probably move the truncate code after the block that removes blank lines, and tags.

**Review by PipoCanaja — CHANGES_REQUESTED** [review]

File: LibreNMS/Alert/Transport/Ciscospark.php (hunk 1 above)
Comment (xorrkaz) [reply LibreNMS/Alert/Transport/Ciscospark.php]: [Face palm].  Yup.  Good call!

**Review by PipoCanaja — APPROVED** [review]
LGTM

**Comment by PipoCanaja** [comment]
Hi @xorrkaz 
Did LibreNMS do the job for this CiscoLiveEMEA ? :)

**Comment by xorrkaz** [comment]
> Hi @xorrkaz 
> 
> Did LibreNMS do the job during this CiscoLiveEMEA as you expected ? :) I did link this PR and PNLNMS-1035 a little bit too late to say hello in Room A10 this morning ... 

Saw that, did you?  Yep.  Worked out very well for us.  As always.

**Pull request merged**